packages/devtools_app/lib/src/screens/vm_developer/vm_service_private_extensions.dart

```diff
@@ -320,11 +318,9 @@ extension FieldPrivateViewExtension on Field {
       guardClassType = json![_guardClassKey]['type'];
     }
 
-    if (guardClassType == '@Class' || guardClassType == 'Class') {
-      return true;
-    } else {
-      return false;
-    }
+    return guardClassType == '@Class' || guardClassType == 'Class'
+        ? true
+        : false;
```

Comment:
could be further simplified to `return guardClassType == '@Class' || guardClassType == 'Class';`

Reply:
Fixed and filed https://github.com/dart-lang/linter/issues/3888